packages/fx-core/src/component/generator/generatorAction.ts

```diff
@@ -116,19 +116,29 @@ export const downloadDirectoryAction: GeneratorAction = {
   },
 };
 
-export const fetchTemplateUrlWithTagAction: GeneratorAction = {
+export const fetchUrlForHotfixOnlyAction: GeneratorAction = {
   name: GeneratorActionName.FetchTemplateUrlWithTag,
   run: async (context: GeneratorContext) => {
     if (context.zip || context.url || context.cancelDownloading) {
       return;
     }
 
     context.logProvider.debug(`Fetching template url with tag: ${JSON.stringify(context)}`);
-    context.url = await fetchTemplateZipUrl(
+    const url = await fetchTemplateZipUrl(
       context.language!,
       context.tryLimits,
       context.timeoutInMs
     );
+    let parts = url.split("/");
+    const selectedTag = parts[parts.length - 2];
+    parts = selectedTag.split(".");
+    // hotfix generates template based on url
+    if (parts[parts.length - 1] !== "0") {
```

Comment:
How about  `if(semver.patch(selectedTag)!==0)`

Reply:
That's a good idea.